src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js

```diff
@@ -0,0 +1,239 @@
+                      fieldName={fieldNames.country}
+                      id="country"
+                      label={`*${_('checkoutProcessForm.contact_information_country')}`}
+                      values={countries}
+                      required
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.zipCode}
+                      id="zipCode"
+                      label={`*${_('checkoutProcessForm.contact_information_zip_code')}`}
+                      withNameValidation
+                      className="field-item--50"
+                    />
+                    <PhoneFieldItem
+                      fieldName={fieldNames.phone}
+                      id="phone"
+                      label={`*${_('checkoutProcessForm.contact_information_phone')}`}
+                      placeholder={_('forms.placeholder_phone')}
+                      className="field-item--50"
+                      required
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.company}
+                      id="company"
+                      label={`${_('checkoutProcessForm.contact_information_company')}`}
+                      className="field-item--50"
+                    />
+                    <InputFieldItem
+                      type="text"
+                      label={`${_('checkoutProcessForm.contact_information_industry')}`}
+                      fieldName={fieldNames.industry}
+                      required
+                      id="industry"
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+            </FieldGroup>
+            {formSubmitted ? (
+              <div className="dp-wrap-confirmation" style={containerMessageSucces}>
+                <div className="dp-msj-confirmation bounceIn" style={flextStartSyle}>
+                  <p>{_('checkoutProcessForm.success_msg')}</p>
+                </div>
+              </div>
+            ) : null}
+            <div className="dp-footer-form" style={flextStartSyle}>
+              <div className="dp-action">
+                <SubmitButton className="dp-button button-medium primary-green">
+                  {_('checkoutProcessForm.save_continue')}
+                </SubmitButton>
+              </div>
+            </div>
+          </fieldset>
+        </FormWithCaptcha>
```

Comment:
Is the captcha useful? In my opinion, it is not necessary in an authenticated session.